mix.exs

```diff
@@ -105,7 +106,9 @@ defmodule Skate.MixProject do
       "assets.setup": ["cmd npm --prefix=assets install"],
       "assets.reset": ["cmd npm --prefix=assets ci"],
       test: ["ecto.create --quiet", "ecto.migrate_all", "test"],
-      fmt: ["format", "cmd npm --prefix=assets run format"]
+      fmt: ["format", "cmd npm --prefix=assets run format"],
+      sobelow:
+        "sobelow --skip --exit=low --ignore Config.HTTPS,Config.CSWH,Config.Headers,Config.CSP"
```

Comment:
Added `Config.Headers` and `Config.CSP` here rather than trying to fix it (and risking "fixing" it wrong) in this PR